docs/docs/recipes/user-profile/README.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 7
+---
+
+# 👤 User Profiles
+
+In previous sections, we demonstrated how to manage your users [using Admin Console](/docs/recipes/manage-users/admin-console), or through [management APIs](/docs/recipes/manage-users/management-api). This is a straightforward process for those in the role of administrator. However, it is also important for the regular users to update their profiles on their own.
+
+To enable this, you will need to create your own profile page with the help of the management APIs.
+
+## Get started
+
+1. Make sure your backend service has acquired a pair of API key and API secret through the [Machine to Machine](/docs/recipes/integrate-logto/machine-to-machine) process.
```

Comment:
Split into different sections?
- Get the management API authorization for your backend server using the machine-to-machine authorization method. 
- Implement your own self-served User Profile management API 
   - example of listed APIs 
   - example of how to leverage Logto-provided management API. i.e /api/verification-code
   - authorize the Logto management API 
   - protect the API
 - Get the User Profile on the Client App
 - Update the User Profile by calling the self-served User Profile management API

Reply:
Makes sense. Will polish the content in a seperate PR
https://linear.app/silverhand/issue/LOG-5306/polish-user-profile-docs